src/app/study-sequence/study-sequence.component.scss

```diff
@@ -1,12 +1,15 @@
 .back-to-view{
-    border-style: dashed;
-    height: 44px;
+    outline: 1px dashed black;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-end: 2;
 }
 
 .container{
     display: grid;
     //Makes a grid with 3 columns
-    grid-template-columns: auto auto auto;
+    grid-template-columns: auto auto 30%;
+    grid-template-rows: auto auto;
     height: calc(100vh - 100px);
```

Comment:
One more change. Since the back to view bar is now a part of the container. The height should be 100vh - 56px.